msglibrary/src/test/java/ingsw/group1/msglibrary/SMSPeerValidityTest.java

```diff
@@ -0,0 +1,81 @@
+package ingsw.group1.msglibrary;
+
+import android.os.Build;
+
+import androidx.annotation.NonNull;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.robolectric.ParameterizedRobolectricTestRunner;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+
+import ingsw.group1.msglibrary.exceptions.InvalidAddressException;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertNotNull;
+
+/**
+ * @author Giorgia Bortoletti
+ * @author Riccardo De Zen
+ */
+@Config(sdk = Build.VERSION_CODES.P)
+@RunWith(Parameterized.class)
+public class SMSPeerValidityTest {
+
+    private static final RandomSMSPeerGenerator GENERATOR = new RandomSMSPeerGenerator();
+    private static final int TEST_RUNS = 200;
+    private final int run;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'run' could be replaced by a local variable.](https://app.codacy.com/manual/RicDeZen/ingsw-group1-message-library/pullRequest?prid=4741620)